scripts/install-worker.sh

```diff
@@ -80,6 +81,9 @@ rm install
 # Remove the ec2-net-utils package, if it's installed. This package interferes with the route setup on the instance.
 if yum list installed | grep ec2-net-utils; then sudo yum remove ec2-net-utils -y -q; fi
 
+# Remove the ec2-net-utils package, if it's installed. This package interferes with the route setup on the instance.
```

Comment:
this id duplicate from above. we shoudl remove

Reply:
removed